src/plugins/biostruct3d_view/src/BioStruct3DGLWidget.cpp

```diff
@@ -245,6 +247,9 @@ void BioStruct3DGLWidget::initializeGL() {
 
 void BioStruct3DGLWidget::resizeGL(int width, int height) {
     glFrame->updateViewPort(width, height);
+    if (lblGlError != nullptr) {
```

Comment:
nit: for the future PRs: we use naming like glFrame (structure type is in the suffix). So lblGlError should be glErrorLabel.